Packs/VMware/ReleaseNotes/1_2_6.json

```diff
@@ -0,0 +1 @@
+{"breakingChanges":true,"breakingChangesNotes": "Added the **redirect_std_out** integration param, use this param when encountering issues with docker timeout."}
```

Comment:
are you sure this needs to be a breaking change? Adding a param that has a default?

Reply:
Yes, because the default is different from what occurs now.